sh/setup-migration-script.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# bash script mode
+set -uo pipefail
+
+# This script creates a migration script to upgrade BHIMA from the previous version of BHIMA
+# to the next version.  It will only pull in data from the next/*.sql folder, so it can only
+# upgrade from the most recent version to the current version.
+
+# NOTE: the file used to determined variables is the .env.development file.  Please make sure
+# this file is up to date.
+
+# TODO(@jniles) - look up current version, and pull in all versions until present version.
+
+echo "[migrate] Migrating BHIMA database"
+
+set -a
+source .env.development
+set +a
```

Comment:
On most of our servers, bhima is running in productioin mode `.env.production`, with this `.env.development` above, we will be oblige to handle migration in production mode in another way than `yarn migrate` (and the mysql DATABASE < MIGRATION_SCRIPT)